app/templates/components/order-card.hbs

```diff
@@ -40,8 +40,8 @@
         {{#if this.order.completedAt}}
           <span>{{t 'on'}} {{general-date this.order.completedAt 'date-time-short'}}</span>
         {{/if}}
-      </span>
-      <div class="extra content small text">
+      </div>
+      <div class="extra content small text pt-2">
```

Comment:
mt-2 would be better
![Screenshot (1)](https://user-images.githubusercontent.com/61330148/104203017-a8711700-5451-11eb-81b1-9a96a88d6e6c.png)